src/bin/pgcopydb/cli_compare.c

```diff
@@ -224,7 +224,6 @@ cli_compare_getopts(int argc, char **argv)
 			{
 				commandline_help(stderr);
 				exit(EXIT_CODE_QUIT);
-				break;
```

Comment:
Can we check that clang is happy with that change? we don't just use gcc.

Reply:
Well I could either build the code from the PR on my own local development machine, or maybe we could use a Dockerfile that uses clang (see https://packages.debian.org/bookworm/clang) and even introduce that as a CI target?

```
$ gcc --version
Apple clang version 14.0.0 (clang-1400.0.29.202)
Target: x86_64-apple-darwin21.6.0
Thread model: posix
```